Mobile Buy SDK/Mobile Buy SDK/Utils/BUYApplePayAuthorizationDelegate.m

```diff
@@ -177,9 +174,39 @@ - (void)updateCheckoutWithAddressCompletion:(void (^)(PKPaymentAuthorizationStat
 		[self.client updateCheckout:self.checkout completion:^(BUYCheckout *checkout, NSError *error) {
 			if (checkout && error == nil) {
 				self.checkout = checkout;
-				[self getShippingRates:self.checkout completion:completion];
-			}
-			else {
+				
+				if ([self.checkout requiresShipping] == NO) {
```

Comment:
I would be a lot happier with this if the completion block was a property of this object, so we didn't need such painful indentation. The logic in here is simply too hard to follow with all the nested conditionals.